.github/workflows/javascript.yml

```diff
@@ -25,14 +25,12 @@ jobs:
       run: |
         bash run-cypress.bash
       env:
-        VITE_PUBLIC_BACKEND_URL: https://tradingstrategy.ai/api
         VITE_PUBLIC_GHOST_API_URL: ${{ secrets.VITE_PUBLIC_GHOST_API_URL }}
         VITE_PUBLIC_GHOST_CONTENT_API_KEY : ${{ secrets.VITE_PUBLIC_GHOST_CONTENT_API_KEY }}
```

Comment:
These and similar vars below can be removed too if they are same as the ones we use in dev